swap_meet/vendor.py

```diff
@@ -1,2 +1,66 @@
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if inventory is None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
+    
+    def get_by_category(self, category, ):
+        items = []
+        for item in self.inventory:
+            if item.category == category:
+                items.append(item)
+        return items
```

Comment:
Nice, nice. This method is a great candidate for using list comprehension if you want to refactor it since our conditional statement we check is pretty simple.

General syntax for list comp:
```python
result_list = [element for element in source_list if some_condition(element)]
```

Which here would look like:

```python
category_items = [item for item in self.inventory if item.category == category]
```

You can also forgo saving the list to the variable category_items and do something like:
```python 
def get_by_category(self, category):
    return [item for item in self.inventory if item.category == category]
```